packages/noya-designsystem/src/components/GridView.tsx

```diff
@@ -0,0 +1,16 @@
+import {
```

Comment:
From our call, example GridView usage might look like:

```ts
import * as GridView from './GridView'

function MyGrid() {
  const items = []

  return (
    <GridView.Root onClick={() => {
      dispatch('selectSwatch', undefined)
    }}>
      {items.map(item => {
        <GridView.Item title={""} subtitle={""} onClick={(event) => {
          event.stopPropagation()
          event.preventDefault()

          dispatch('selectSwatch', item.do_objectID)
        }}>{children}</GridView.Item>
      })}
    </GridView.Root>
  )
}
```

So we should export a `Root` and an `Item` component from this file.

Reply:
It  uses the `GridView` now instead of the awkward events. Although the preventDefault it's not really working. I'm not sure if I am doing something wrong.